lib/functions_framework.rb

```diff
@@ -177,14 +177,22 @@ def cloud_event name = DEFAULT_TARGET, &block
     # look up the given target function name in the global registry.
     #
     # @param target [String] The name of the function to run
+    # @param assert_signature_type ["http","cloudevent",nil] Optional. If
```

Comment:
- I would assume `signature_type` would be the name.
- I would assume that all arguments may have assertions and that we shouldn't need to change the variable name if the function config value is asserted or not.
- We do not use such a name with `target` even though we raise a similar arg error for undefined functions.

Reply:
I used `assert_signature_type` to make it clear that it in fact is _not setting_ the signature type (unlike `target` which _sets_ the target). But hmm, you're probably right, the distinction is not really worth it. Changing back to `signature_type`.

Reply:
I ended up ditching this parameter altogether, and just moving the signature type verification out into the CLI class. It is arguably just a function of the CLI and doesn't belong in the framework core in the first place.